.gitignore

```
@@ -14,7 +14,8 @@ website/node_modules/
 website/tech-doc-hugo
 
 # Ensure that will not commit the bin gen in the go sample
-testdata/go/memcached-operator/bin/*
+testdata/go/v2/memcached-operator/bin/*
+testdata/go/v3/memcached-operator/bin/*
```

We should combine all `bin/` ignores
```suggestion
*/bin/*
```